schema/ContainerImage.schema.yaml

```diff
@@ -0,0 +1,102 @@
+title: ContainerImage
+'@id': stencila:ContainerImage
+extends: Thing
+role: primary
+status: experimental
+category: code
+description: A linux container image (such as provided by Docker or Singularity) that is used in scientific computing.
+properties:
+  annotations:
+    '@id': stencila:annotations
+    description: | 
+      One or more annotations (labels) for the container image, typically key value pairs represented as strings
+    aliases:
+      - labels
+    type: array
+    items:
+      $ref: Mark
+  applicationCategory:
+    '@id': stencila:applicationCategory
+    description: The container technology (e.g., Docker or Singularity)
+    type: string
```

Comment:
According to https://schema.org/applicationCategory this property name is for "Type of software application, e.g. 'Game, Multimedia'.". I wonder if https://schema.org/runtimePlatform would be more appropriate here?

Reply:
Definitely agree! Is there a way to essentially hard code a default applicationCategory (and we would call it linux containerization?) or is that not necessary?

Reply:
Yes, you can use the `const` keyword e.g. `const: Linux containerization`